README.md

```diff
@@ -93,7 +95,7 @@ The `docs` directory has some user documentation written in [AsciiDoc](https://d
 You can render it to HTML using:
 
 ```
-./mvnw org.asciidoctor:asciidoctor-maven-plugin:process-asciidoc@convert-to-html
```

Comment:
`mvnw` isn't a typo and should still work. Did it fail for you? 

mvnw is invoking maven wrapper which will install (?)/run desired maven version even if not installed locally.

Reply:
mvnw works, this is just the only usage of it compared to 7 `mvn` usages in README.md